Cython/Compiler/Nodes.py

```diff
@@ -2721,6 +2724,9 @@ def generate_function_header(self, code, with_pymethdef, with_opt_args=1, with_d
             code.globalstate.parts['module_declarations'].putln(
                 "%s%s%s; /* proto*/" % (storage_class, modifiers, header))
         code.putln("%s%s%s {" % (storage_class, modifiers, header))
+        # silence unused warnings when CYTHON_UNUSED does not work (e.g. on msvc)
+        for arg in unused_args:
+            code.putln("CYTHON_UNUSED_VAR(%s);" % arg)
```

Comment:
How can I fix this for C89? I currently simply generate them directly behind the opening brackets. However there could be variable declarations afterwards.